webapp/src/js/enketo/widgets/phone-widget.js

```diff
@@ -92,7 +83,7 @@ define( function( require, exports, module ) {
     // TODO(estellecomment): move this to a catch clause, when settings aren't found.
     formatAndCopy( $proxyInput, $input, {} );
 
-    this.builtPromise = Settings()
+    this.builtPromise = Settings.get()
```

Comment:
It will always be available, because it's injected in the main app component, which is available before any other component is rendered. All this javascript is executed AFTER view init of any component.